packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -3,6 +3,8 @@ Table
 
 Tables show tabular data in columns and rows.
 
+@status Beta
```

Comment:
Let's change this to `@status Draft` https://design.cms.gov/guidelines/component-maturity/